Metalama.Testing.UnitTesting/UnitTestClass.cs

```diff
@@ -34,6 +35,8 @@ protected UnitTestClass( ITestOutputHelper? testOutputHelper = null, bool inject
             this._injectLoggingService = injectLoggingService;
         }
 
+        public virtual void Dispose() { }
```

Comment:
Implement the IDisposable pattern properly (with `protected virtual void Dispose(bool)`)

Reply:
I think the full dispose pattern should not be used here. You'd always need to have `if (disposing) { ... }` because if you throw from a finalizer of the test class (by assertion), the whole test host process will crash. I've moved IDisposable implementation to the test class that needs it, there is no need to generalize this.

Reply:
Ah sorry I beleived it was private but if it's internal it's okay.